clang/test/CIR/CodeGen/asm.c

```diff
@@ -1,32 +1,32 @@
 // RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
 // RUN: FileCheck --input-file=%t.cir %s
 
-//CHECK: cir.asm(x86_att, {"" "~{dirflag},~{fpsr},~{flags}"})  : () -> ()
+//CHECK: cir.asm(has_side_effects, x86_att, {"" "~{dirflag},~{fpsr},~{flags}"})  : () -> ()
```

Comment:
Looks like you are missing test coverage for `is_align_stack`.

Reply:
I just removed `stack_align` from `cir.asm` since it equals to `false`